packages/data-stores/src/starter-designs-queries/use-starter-designs-generated-query.ts

```diff
@@ -0,0 +1,34 @@
+import { useQuery, UseQueryResult } from 'react-query';
+import wpcomRequest from 'wpcom-proxy-request';
+import type { StarterDesignsGenerated } from './types';
+import type { Design } from '@automattic/design-picker/src/types';
+
+export function useStarterDesignsGeneratedQuery(): UseQueryResult< Design[] > {
+	return useQuery( [ 'starter-designs-generated' ], () => fetchStarterDesignsGenerated(), {
+		select: ( response ) => response.map( apiStarterDesignsGeneratedToDesign ),
+		enabled: true,
+		staleTime: Infinity,
```

Comment:
I think we have to set something like `refetchOnMount: always`. Otherwise, the cache won't expire.

Reply:
I'm sure I'm missing something here, but when do we need to refetch?

Reply:
If we create/update the recipes, I think we have to get the latest ones. Does it make sense?